lib/src/default_extension_map.dart

```diff
@@ -434,6 +434,8 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'mc1': 'application/vnd.medcalcdata',
   'mcd': 'application/vnd.mcd',
   'mcurl': 'text/vnd.curl.mcurl',
+  // https://www.rfc-editor.org/rfc/rfc7763
```

Comment:
The RFS also mentions `.markdown` as an extension. Do we think that's unpopular enough now to not be worth supporting?

Reply:
Good catch! In 2014, one of the authors of Markdown actually [endorsed](https://web.archive.org/web/20200712120733/https://daringfireball.net/linked/2014/01/08/markdown-extension) `.markdown` as the only extension they would use. Either way, I think it makes sense to add for the sake of completeness. I'll prepare a commit.